### PR TITLE
feat(lib): Applies license and copyright headers

### LINKED ISSUE #423: Make licenses for the SampleServer and Libs clearer
Apply license information according MPL-2.0 to the respective files.

### PR DESCRIPTION
**Pull request by GoetzGoerisch**
Fixes #423 

Co-authored-by:  Christian von Arnim <christian.von-arnim@isw.uni-stuttgart.de>
Co-authored-by: Tonja Heinemann <tonja.heinemann@isw.uni-stuttgart.de>
Co-authored-by: Christoph Ruckstetter <christoph.ruckstetter@weinig.com>
Co-authored-by: Dennis Hezel <dh@3yourmind.com>
Co-authored-by: Hermann Leinweber <79450466+basyskom-hlei@users.noreply.github.com>

### CODE REVIEW

File: MachineTools/MRMachineTool.cpp [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright 2020-2021 (c) Christian von Arnim, ISW University of Stuttgart (for umati and VDW e.V.)
+ */
```
Comment (TonjaHeinemann) [inline MachineTools/MRMachineTool.cpp:6]: I authored the original version of this file in 2020.
Suggestion: add resp. Copyright note.

File: MachineTools/MRMachineTool.hpp [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright 2020-2021 (c) Christian von Arnim, ISW University of Stuttgart (for umati and VDW e.V.)
+ */
```
Comment (TonjaHeinemann) [inline MachineTools/MRMachineTool.hpp:6]: I authored the original version of this file in 2020.
Suggestion: add resp. Copyright note.

**Review by TonjaHeinemann — CHANGES_REQUESTED** [review]
Add me in copyright notice for MRMachineTool (I authored the original version on the ISW Git)

File: MachineTools/MRMachineTool.cpp (hunk 1 above)
Comment (GoetzGoerisch) [reply MachineTools/MRMachineTool.cpp:7]: Overlooked, fixed!

File: MachineTools/MRMachineTool.hpp (hunk 2 above)
Comment (GoetzGoerisch) [reply MachineTools/MRMachineTool.hpp:7]: Overlooked, fixed!

**Comment by GoetzGoerisch** [comment]
Thank you @TonjaHeinemann for the comments. I overlooked history of the files in the first place. Fixed now.

**Comment by IOTeule** [comment]
Dear All, this is really a GREAT step, thx a lot for your effort to clarify and apply different licenses. 
Some open points IMO:

1) CMakeLists.txt no license in repro (should be MIT)
2) CMakeLists.txt no license in .github  (used by initialbuild.sh)
A second initialBuildServer.sh could be solution with pointing to a folder .server (both complete MIT)
So build step is a two step process a) initialbuild.sh (remove server) b) initialBuildServer.sh (server including lib)
3) There should be at least a very basic machine under MIT license
4) some config files (e.g. .gitpod.Dockerfile )

As written in https://github.com/umati/Sample-Server/issues/423 I would prefer a clear MIT server.

**Comment by ccvca** [comment]
Looks good, maybe MIT can be added here: cmake/FindReflCpp.cmake
As I have no easy to use solution for finding files before 2020, I suspecpretend it should be fine.

PS: As open62541 is not MIT, there can't be a fully MIT-licenced server.

**Review by ccvca — APPROVED** [review]

**Comment by GoetzGoerisch** [comment]
@TonjaHeinemann @ChristophRuckstetter could you please review?

**Review by TonjaHeinemann — APPROVED** [review]

**Review by ChristophRuckstetter — APPROVED** [review]

**Pull request merged**